src/static/css/pages/sub-pages.less

```diff
@@ -3,10 +3,51 @@
    Sub-Pages pages at /sub-pages/*.
    ========================================================================== */
 
+.green-bullets {
+    li {
+        color: @black;
+        list-style: none;
+    }
+
+    li:before {
+        position: relative;
+        top: .1em;
+        margin-left: -1em;
+        padding-right: .3em;
+
+        color: @green;
+        //Square bullet point
+        content: "▪ ";
+        font-size: 1.5em;
+
```

Comment:
> Should we use the entity code for the content?

I was wondering about this. I had a tricky time finding the code for small square.